### DISCUSSION TITLE
GeigerLog: Connection problems

### DISCUSSION THREAD

**Discussion by alfmck** [Gissio/radpro#77] [Q&A]
When I tried to connect to device in GeigerLog soft, my Win11 hangs. I solved this by switching off Bluetooth device function.
Now my GC-01 reliable connects to GeigerLog using standard COM driver, while FS5000 for connection uses CH 340 driver.
Would be interesting to know where the problem is?

**Comment by Gissio** [comment]
I think you should post this request in the GeigerLog forum!

**Reply by alfmck** [reply]
It isn't for GeigerLog only, because F5000 with its remote control soft has common problem. I suspect that something wrong with the mechanism of distributing usb ports between COM ports. Or its in mine PC only. Simply I wanted to warn radpro users.
